readme.md

```diff
@@ -165,12 +165,13 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
   1. Check out the repository, this can be done using [GitHub Desktop](https://desktop.github.com) or [other tools](https://help.github.com/articles/which-remote-url-should-i-use)
   2. Open a new Developer Command Prompt for VS 2022
   3. Navigate to the repository (e.g. `cd C:\GitHub\OpenRCT2`)
-  4. To build the 64-bit version, use `msbuild openrct2.proj /t:build /p:platform=x64`
-
-     To build the 32-bit version, use `msbuild openrct2.proj /t:build /p:platform=Win32`
+  4. To build the x64 version, use `msbuild openrct2.proj /t:build /p:platform=x64`
+     To build the x86 version, use `msbuild openrct2.proj /t:build /p:platform=Win32`
+     To build the Arm64 version, use `msbuild openrct2.proj /t:build /p:platform=arm64`
```

Comment:
Shouldn't it be `ARM64` according to https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-target-framework-and-target-platform?view=vs-2022#target-platform?

Reply:
These names reflect the used platform configurations in `openrct2-win.vcxproj` (https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2-win/openrct2-win.vcxproj#L69C7-L69C85). Either way, it doesn't matter because `msbuild` is not case-sensitive for properties.